pandas/tests/series/test_ufunc.py

```diff
@@ -31,17 +31,17 @@ def arrays_for_binary_ufunc():
 @pytest.mark.parametrize("sparse", SPARSE, ids=SPARSE_IDS)
 def test_unary_ufunc(ufunc, sparse):
     # Test that ufunc(pd.Series) == pd.Series(ufunc)
-    array = np.random.randint(0, 10, 10, dtype="int64")
-    array[::2] = 0
+    u_arr = np.random.randint(0, 10, 10, dtype="int64")
```

Comment:
can't this be `arr`? I just tried it and it was fine - besides, what does `u_arr` stand for? Unary array? If so, what is that?